examples/event-sources/mqtt.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       # source will be JSON
       jsonBody: true
       # client id
-      clientId: 2345
+      clientId: "2345"
```

Comment:
Could you correct the commented example as well?

Reply:
done